client/src/screens/menu/view.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+import { Layout, Navbar } from '../../components';
+import { connect } from '../../utils';
+import { controller } from './controller';
+
+class PureMenu extends React.Component {
+  render() {
+    return (
+      <Layout bottomBar={<Navbar />}>
+        <div style={{ textAlign: 'center' }}>
+          Menu
+        </div>
+      </Layout>
+    );
+  }
+}
+
+export const Menu = connect({
+  controller,
+}, ctrl => ({
+}))(PureMenu);
```

Comment:
function below returns empty object, which means that this code does nothing (returns no data, is useless).
```
ctrl => ({
})
```
delete these lines (19-22), and change line 7 as below:
`export class Menu extends React.Component {`